tests/test_operation.py

```diff
@@ -13,6 +13,12 @@ def test_operation(accounts, token, vault, strategy, strategist, amount, user, c
 
     # harvest
     crv3.transfer(strategy, 10e20, {"from": whale_3crv})
+
+    # Done to fix the UniswapV2: K issue
+    pairs = [strategy.ethCrvPair(), strategy.ethYveCrvPair(), strategy.ethUsdcPair()]
+    for pair in pairs:
+        Contract(pair, owner=strategist).sync()
```

Comment:
@TheRyanMiller @andy8052 this fixes the K issue in the test but now I get

<img width="694" alt="Screen Shot 2021-03-25 at 10 17 17 AM" src="https://user-images.githubusercontent.com/78830419/112487659-e68c9000-8d4a-11eb-9e9d-79e9434209fd.png">

plz verify the test